x-pack/plugins/cases/public/components/case_view/does_not_exist.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { EuiEmptyPrompt, EuiButton } from '@elastic/eui';
+import * as i18n from './translations';
+import { CasesNavigation } from '../links';
+interface Props {
+  allCasesNavigation: CasesNavigation;
+  caseId: string;
+}
+export const DoesNotExist = ({ allCasesNavigation, caseId }: Props) => (
+  <EuiEmptyPrompt
+    // @ts-ignore remove once https://github.com/elastic/eui/issues/4850 is resolved
+    iconColor={null}
```

Comment:
Per https://github.com/elastic/eui/issues/4850#issuecomment-853326052 you need to pass `default`.